src/main/ui/app/route/SearchPage.jsx

```diff
@@ -33,7 +33,8 @@ export default class SearchPage extends Component {
     doSearch() {
         const query = this.props.location.query.q;
         const type = this.props.location.query.type;
-        const searchKey = `type:${type ? type : "all"} query:${query ? query : ""}`;
+	const tag = this.props.location.query.tag;
```

Comment:
This does have the same formatting. Actually it's showing it incorrectly here for some reason i.e. if you click on the above view, the formatting is fine... Strange but true :)